src/Shouldly/ShouldStaticClasses/ShouldThrowExtensionsOverloads.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace Shouldly
+{
+    public static partial class Should
```

Comment:
It looks like updating the extension to return `Should.Throw` will result in the error message changing from "action should throw" to "return Should throw". It'd be nice to keep the variable (`action`) in the message. Is it worth looking at a custom message generator or do you point me in the direction of another way around this problem that allows us to move these methods into their own class yet retain the helpful error message?